server/game/cards/05-DT/Mookling.js

```diff
@@ -0,0 +1,14 @@
+const Card = require('../../Card.js');
+
+class Mookling extends Card {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'opponent',
+            effect: ability.effects.modifyKeyCost(() => this.getPower(false))
```

Comment:
this is tricky, should avoid using "this"

Reply:
So, I tried to avoid using it, but it seems that "context.source" is undefined. Example, this fails:

`effect: ability.effects.modifyKeyCost((context) => context.source.getPower(false))`

I looked at other cards using persistentEffect and they also use "this" to find the card with the actual effect. Maybe A) I'm missing something, or B) there is a something up which is not assigning context.source to be the card with the effect?

Reply:
I'll keep Mookling around with it.

Reply:
Figured it out.. though I have no idea why this works:
`effect: ability.effects.modifyKeyCost((_, context) => context.source.getPower(false))`